README.md

```diff
@@ -59,3 +59,35 @@ and this will start a development server.
 
 On Android, use the Expo app to scan the QR code from your terminal to open your project.
 On iOS, use the built-in QR code scanner of the Camera app.
+
+
+
+## Run Cypress tests
+
+To test our application we can use Cypress. The first step is to open our IDE with the application folder and run the following command in the terminal:
+
+```
+npx expo install react-native-web@~0.18.9 react-dom@18.1.0 @expo/webpack-config@^0.17.2npm
+```
```

Comment:
This part should be removed. Those items will be installed at the first `npm install`, already defined on another section of the README